samples/subsys/usb/cdc_acm_bridge/README.rst

```diff
@@ -0,0 +1,50 @@
+.. zephyr:code-sample:: usb-cdc-acm-bridge
+   :name: USB CDC-ACM bridge
+
```

Comment:
Add so this shows up on the UART doc page as a relevant sample
```
   :relevant-api: uart_interface
```

Reply:
done